tests/integration/s2n_client_endpoint_handshake_test.py

```diff
@@ -176,7 +182,9 @@ def well_known_endpoints_test(use_corked_io, tls13_enabled, fips_mode):
 
         # Retry handshake in case there are any problems going over the internet
         for i in range(1, maxRetries):
+            print("Connecting to: %-35sAttempt: %-10s... " % (endpoint, i))
             ret = try_client_handshake(endpoint, arguments, expected_cipher)
```

Comment:
This does clutter the output a little:
```
Connecting to: www.akamai.com                     Attempt: 1         ... 
Endpoint: www.akamai.com                     Expected Cipher: Any                                     ... PASSED
Connecting to: www.amazon.com                     Attempt: 1         ... 
Endpoint: www.amazon.com                     Expected Cipher: Any                                     ... PASSED
Connecting to: s3.us-west-2.amazonaws.com         Attempt: 1         ... 
Endpoint: s3.us-west-2.amazonaws.com         Expected Cipher: Any                                     ... PASSED
```

Maybe only print this on attempts after the first? So only on retries?

Reply:
That sounds fine. I am inclined to merge this now so we can unblock other PRs (since CI times take up to 2hr). And then I can open another PR to amend this.

Reply:
PR for the update: https://github.com/aws/s2n-tls/pull/3151